### DISCUSSION TITLE
Super easy to use 👍

### DISCUSSION THREAD

**Discussion by euvl** [rpearce/react-medium-image-zoom#294] [General]
Hey guys,

Just wanted to tell you that the library is great, was very easy to integrate, took me literary a minute to just make it work within our prosemirror/tiptap editor. Even perfectly works for `divs` with image backgrounds (aka Instagram-style photos).

Great work 👍

**Comment by rpearce** [comment]
@euvl You are super kind for sharing this, and it makes my day 💯 better.

The next major version is getting there ([here is an update](https://github.com/rpearce/image-zoom/discussions/274#discussioncomment-1616865)), so stay tuned.
